File: Java/gesbrasilburger/src/main/java/sn/brasilburger/entity/Livreur.java
package sn.brasilburger.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Livreur {
    private int id;
    private String nomComplet;
    private String telephone;
    private Zone zone;
    private boolean etat = true ;
    @Override
    public String toString(){
        return " Livreur->[ID: " + id + ", Nom Complet: " + nomComplet + ", Telephone: " + 
        telephone + ", Zone: " + zone.getNom() + "]";
    }
}

File: CSharp/brasilBurger/Models/Livreur.cs
namespace brasilBurger.Models
{
    public class Livreur
    {
        public int Id { get; set; }
        public string NomComplet { get; set; }
        public string Telephone { get; set; }
        public bool Etat { get; set; }

        public int ZoneId { get; set; }
        public Zone Zone { get; set; }
        public ICollection<Commande> Commandes { get; set; } = new List<Commande>();
    }
}